src/SFA.DAS.EmployerAccounts.Web/Helpers/HtmlHelperExtensions.cs

```diff
@@ -22,13 +24,44 @@ public static bool IsAuthorized(this HtmlHelper htmlHelper, string featureType)
             return isAuthorized;
         }
 
+        public static MvcHtmlString RenderReturnToHomePageButton(this HtmlHelper htmlHelper, string accountId)
+        { 
+           var tier2User = htmlHelper.ViewContext.RequestContext.HttpContext.User?.IsInRole(Tier2User);
+           bool IsTier2User = tier2User ?? false;
+           bool IsAccountIdSet = accountId != null;
+
+            var homePageButton = $"<a class=\"button\" href=\"";              
+            homePageButton += IsTier2User && IsAccountIdSet ? $"accounts/" + accountId + "/teams/view\"" : (IsAccountIdSet ? $"accounts/" + accountId + "/teams\"" : $"/\"");
+            homePageButton += IsTier2User && IsAccountIdSet ? $">Return to your team </a>" : (IsAccountIdSet ? $">Go back to the account home page</a>" : $">Go back to the service home page</a>");           
+            
+            return MvcHtmlString.Create(homePageButton);
+        }
+
+
+        public static MvcHtmlString RenderReturnToHomePageLinkForBreadcrumbSection(this HtmlHelper htmlHelper, string accountId)
```

Comment:
Nothing in this method to indicate it's breadcrumb specific.  Maybe drop the breadcrumb from the name, it could be used elsewhere.